public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -163,30 +155,36 @@ function FolderGroupAndEvaluationInterval({
 
   const onOpenEditGroupModal = () => setIsEditingGroup(true);
 
-  const editGroupDisabled = groupfoldersForGrafana?.loading || isNewGroup || !folder || !group;
+  const editGroupDisabled = groupfoldersForGrafana?.loading || isNewGroup || !folderName || !groupName;
+
+  const emptyNamespace: CombinedRuleNamespace = {
+    name: folderName,
+    rulesSource: GRAFANA_RULES_SOURCE_NAME,
+    groups: [],
+  };
+  const emptyGroup: CombinedRuleGroup = { name: groupName, interval: evaluateEvery, rules: [] };
 
   return (
     <div>
       <FolderAndGroup initialFolder={initialFolder} />
-      {isEditingGroup && (
+      {folderName && isEditingGroup && (
         <EditCloudGroupModal
-          groupInterval={evaluateEvery}
-          nameSpaceAndGroup={{ namespace: folder?.title ?? '', group: group }}
-          sourceName={GRAFANA_RULES_SOURCE_NAME}
+          namespace={existingNamespace ?? emptyNamespace}
+          group={existingGroup ?? emptyGroup}
           onClose={() => closeEditGroupModal()}
-          folderAndGroupReadOnly
+          intervalEditOnly
```

Comment:
nit: not a big fan of this but it makes sense for now – perhaps if we start adding more disabled fields we can introduce an array of field names that should be disabled

Reply:
I'm not a fan either :) This is due to using the same modal in two places. Perhaps the new rule creation flow won't use this modal so it would be used only once on the alert list view